csrankings-j.csv

```diff
@@ -1747,7 +1747,6 @@ Jörg Henkel,Karlsruhe Institute of Technology,http://ces.itec.kit.edu/21_henkel
 Jörg Hoffmann 0001,Saarland University,http://fai.cs.uni-saarland.de/hoffmann,NR6qg_UAAAAJ
 Jörg Kienzle,McGill University,http://www.cs.mcgill.ca/~joerg/Home/Jorgs_Home.html,INUL3eEAAAAJ
 Jörg Liebeherr,University of Toronto,http://www.comm.utoronto.ca/~jorg,8WrYVG8AAAAJ
-Jörg Müller,Aarhus University,http://pure.au.dk/portal/en/persons/id(3e718f2c-7cc4-4cc5-8f05-87e65963d7c1).html,qU5PVzAAAAAJ

```

Comment:
Please restore but update affiliation and web page to University of Bayreuth, http://joergmueller.info/.

Reply:
This still has not been done, @amoeller .

Reply:
He is already there (as Jörg Müller 0001).